components/bigcommerce/sources/new-customer/new-customer.mjs

```diff
@@ -0,0 +1,26 @@
+import common from "../common/base.mjs";
+
+export default {
+  ...common,
+  name: "New Customer",
+  version: "0.0.1",
+  type: "source",
+  key: "bigcommerce-new-customer",
+  description: "Emit new created customer",
+  hooks: {
+    ...common.hooks,
+    async activate() {
+      const hookId = await this.bigcommerce.createHook(
+        this.http.endpoint,
+        "customer",
+      );
+      this.setHookId(hookId);
+    },
+  },
+  methods: {
+    ...common.methods,
+    getSummary() {
+      return "Customer created";
+    },
+  },
```

Comment:
I fixed the PR title, they are triggers, not actions.
This trigger receives webhooks from customers created, so we don't have the customer's id until we receive

Reply:
Hi, I guess my suggestion was unclear, sorry for that. What I meant is to retrieve other customer information with the id provided from webhook event. It would be great to retrieve other customer information and emit it while emitting the event. I think emitting the customer info instead of just customer id would be more helpful to users.

Reply:
I agree with you
but I think that if a user wants more information, it's possible to link to the next action that does that

Reply:
I already doing that in this issue
https://github.com/PipedreamHQ/pipedream/issues/2862